sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/ImportJobDigitalTwinOptions.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.Context;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The optional parameters for
+ * {@link com.azure.digitaltwins.core.DigitalTwinsClient#listImportJobs(ImportJobDigitalTwinOptions, Context)} and
+ * {@link com.azure.digitaltwins.core.DigitalTwinsAsyncClient#listImportJobs(ImportJobDigitalTwinOptions)}
+ */
+@Fluent
+public final class ImportJobDigitalTwinOptions {
+    /*
+     * The maximum number of items to retrieve per request. The server may
+     * choose to return less than the requested number.
+     */
+    @JsonProperty(value = "MaxItemsPerPage")
```

Comment:
FYI - looking at the service code, it doesn't currently support a max-items-per-page setting. The swagger does, however, define it. Also, the header key is "max-items-per-page", not "MaxItemsPerPage".

Reply:
I followed the pattern used by other classes and it used MaxItemsPerPage.
![image](https://user-images.githubusercontent.com/114272783/218791479-6803aa73-6023-4a5d-a9b1-9a766cf03cfb.png)

However, when we call the list API, it passes the expected header parameters:
![image](https://user-images.githubusercontent.com/114272783/218791270-a9c07b07-893b-4107-80ff-1bd864733f72.png)
Let me know if you would like me to update it.

Reply:
Ah, I see - thanks for clarifying! That's a bit confusing. But good to see you're following the pattern here.